test/smtcallback.js

```diff
@@ -146,6 +151,11 @@ tape('SMTCheckerCallback', function (t) {
         continue;
       }
 
+      let ignoreCounterexamples = false;
```

Comment:
If you use `ignoreCex` above and in the test sources, perhaps use the same here?

Reply:
I wanted to, but in the `test` object it's also called `ignoreCex`, I wasn't sure about shadowing in the assignment below